connection.go

```diff
@@ -104,6 +111,16 @@ func (ctn *Connection) IsConnected() bool {
 	return ctn.conn != nil
 }
 
+// IsFresh returns true if the connection has not yet reached the idle deadline.
+func (ctn *Connection) IsFresh() bool {
```

Comment:
It doesn't have to. I just did it because there is ```IsConnected``` but yeah, since the one controlling this is node, not connection, it doesn't have to be public.